package-lock.json

```diff
@@ -9,189 +9,129 @@
       "version": "0.1.0",
       "license": "MIT",
       "dependencies": {
-        "bootstrap": "^4.4.1",
-        "htmx.org": "^1.8.5",
-        "jquery": "^3.6.0",
-        "popper.js": "^1.14.3"
+        "htmx.org": "^1.9.12",
+        "hyperscript.org": "^0.9.12"
```

Comment:
```suggestion
```

Reply:
Thanks for pointing that out. I hadn't re-run the install step after removing it. This file is automatically generated so best to let the generator do its thing.